sympy/printing/pretty/pretty.py

```diff
@@ -1715,6 +1715,9 @@ def _print_PolyRing(self, ring):
     def _print_FracField(self, field):
         return prettyForm(sstr(field))
 
+    def _print_FreeGroupElm(self, elm):
+        return prettyForm(str(elm))
```

Comment:
Actual pretty printing would be nice. But for that, you may need to be using Expr or Basic subclasses.